packages/hash/playwright/tests/entity-creation.spec.ts

```diff
@@ -23,8 +23,8 @@ test("user can create and update entity", async ({ page }) => {
   // Fill up entity creation form
   await page.click('text=NameDescription >> input[type="text"]');
   await page.fill('text=NameDescription >> input[type="text"]', entityName);
-  await page.click(':nth-match(input[type="text"], 3)');
-  await page.fill(':nth-match(input[type="text"], 3)', "Test Entity");
+  await page.click(':nth-match(input[type="text"], 2)');
+  await page.fill(':nth-match(input[type="text"], 2)', "Test Entity");
```

Comment:
Hmm interesting and makes sense! If tests pass, how about making this selector less page-aware? We can select some screen area first and then an 'nth' input. Tests will get less fragile.

Reply:
yeah I was thinking the same. Not sure what's the best way to do this in playwright. Should we give an `id` to the text field in the html so that we can reliably get the input from its id or is there a canonical way of doing this predictably in playwright?

Reply:
```suggestion
  await page.click('main :nth-match(input[type="text"], 2)');
  await page.fill('main :nth-match(input[type="text"], 2)', "Test Entity");
```

![Screenshot 2022-01-18 at 18 15 12](https://user-images.githubusercontent.com/608862/149995211-ab4cbf69-75c6-4ba9-8562-97a51a747361.png)

This will keep the tests working when we bring back the input. I haven't checked new selectors – my suggestion is just a guess.

Reply:
oh I see what you mean, that makes more sense. I'll try it this way